### DISCUSSION TITLE
Dynamic Collections

### DISCUSSION THREAD

**Discussion by SlowLogicBoy** [ppy/osu#13010] [Ideas]
I like to use map filter extensively in osu! for example:
`AR>=9 Stars<6`

I would like to save these as collections, and when I get new maps or delete them they would update automatically (probably execute that query once I press on a collection).

Or there could be another approach to save those queries in a search dropdown list.

**Comment by peppy** [comment]
Definitely something we want in lazer.

**Comment by luaneko** [comment]
The filtering part is being done in my pr #3906
